playground/backend/internal/environment/environment_service.go

```diff
@@ -177,13 +172,9 @@ func createExecutorConfig(apacheBeamSdk pb.Sdk, configPath string) (*ExecutorCon
 	}
 	switch apacheBeamSdk {
 	case pb.Sdk_SDK_JAVA:
-		executorConfig.CompileArgs = append(executorConfig.CompileArgs, getEnv(beamPathKey, defaultBeamSdkPath))
-		jars := strings.Join([]string{
-			getEnv(beamPathKey, defaultBeamSdkPath),
-			getEnv(beamRunnerKey, defaultBeamRunner),
-			getEnv(SLF4jKey, defaultSLF4j),
-		}, ":")
```

Comment:
Don't we need all these configs?

Reply:
As I understand they all are included here `DefaultBeamJarsPath `

Reply:
Aydar is right